analysis/druidrestoration/src/modules/features/PrematureRejuvenations.tsx

```diff
@@ -28,92 +36,123 @@ class PrematureRejuvenations extends Analyzer {
   static dependencies = {
     healingDone: HealingDone,
     combatants: Combatants,
+    hotTracker: HotTrackerRestoDruid,
```

Comment:
Refactored this class to use the the more detailed info from `HotTrackerRestoDruid` rather than trying to generate it itself.